docs/approach.rst

```diff
@@ -62,23 +62,28 @@ value for that voxel in the adaptive mask.
 Monoexponential decay model fit
 ```````````````````````````````
 The next step is to fit a monoexponential decay model to the data in order to
-estimate voxel-wise :math:`T_{2}^*` and :math:`S_0`.
+estimate voxel-wise :math:`T_{2}^*` and :math:`S_0`. :math:`S_0` corresponds
+to the total signal in each voxel before decay and can reflect coil sensivity. 
+:math:`T_{2}^*` corresponds to the rate at which a voxel decays over time, which
+is related to signal dropout and BOLD sensitivity. Estimates of the parameters are 
+saved as **t2sv.nii.gz** and **s0v.nii.gz**. 
+
 While :math:`T_{2}^*` and :math:`S_0` in fact fluctuate over time, estimating
 them on a volume-by-volume basis with only a small number of echoes is not
 feasible (i.e., the estimates would be extremely noisy).
 As such, we estimate average :math:`T_{2}^*` and :math:`S_0` maps and use those
 throughout the workflow.
 
 In order to make it easier to fit the decay model to the data, ``tedana``
-transforms the data.
+transforms the data by default.
```

Comment:
Is there any way to turn this off ? `by default` makes me think there's another option.

Reply:
the monoexponential fit doesn't transform the data. I've now added a note below about that.